wapitiCore/attack/mod_csp.py

```diff
@@ -0,0 +1,102 @@
+import re
+
+from wapitiCore.attack.attack import Attack
+from wapitiCore.net.web import Request
+from wapitiCore.language.vulnerability import Additional, _
+
+
+# This module check the basics recommendations of CSP
+class mod_csp(Attack):
+    name = "csp"
+
+    check_list = ["default-src", "script-src", "object-src", "base-uri"]
+
+    check_default = ["*", "unsafe-eval"]
+    check_script_src = ["unsafe-inline", "data:", "http", "https"]
+    check_object_src = ["none"]
+    check_base_uri = ["none", "self"]
+
+    all_check = [check_default, check_script_src, check_object_src, check_base_uri]
+
+    def check_policy_values(self, policy_name, check_list, csp_dict):
```

Comment:
Prototype of this function is name because it will be easy to write tests (and we need them)